File: src/gestorAplicacion/Veterinaria/Turno.java
package gestorAplicacion.Veterinaria;

import java.io.Serializable;
import gestorAplicacion.Clientes.Animal;
import gestorAplicacion.Clientes.Cliente;
import gestorAplicacion.Clientes.Mascota;

public class Turno implements Serializable{
	
	/**
	 * 
	 */
	private static final long serialVersionUID = 5L;
	private int horaInicio;
	private String fecha;
	private Animal mascota;
	private Cliente cliente;
	private boolean disponibilidad;
	private Persona veterinario;
	
	
	public Turno(int horaInicio, String fecha, Animal mascota, Cliente cliente, boolean disponibilidad,
			Persona veterinario) {
		super();
		this.horaInicio = horaInicio;
		this.fecha = fecha;
		this.mascota = mascota;
		this.cliente = cliente;
		this.disponibilidad = disponibilidad;
		this.veterinario = veterinario;
	}
	
	
	public int getHoraInicio() {
		return horaInicio;
	}
	public void setHoraInicio(int horaInicio) {
		this.horaInicio = horaInicio;
	}
	public String getFecha() {
		return fecha;
	}
	public void setFecha(String fecha) {
		this.fecha = fecha;
	}
	public Animal getMascota() {
		return mascota;
	}
	public void setMascota(Animal mascota) {
		this.mascota = mascota;
	}
	public Cliente getCliente() {
		return cliente;
	}
	public void setCliente(Cliente cliente) {
		this.cliente = cliente;
	}
	public boolean isDisponibilidad() {
		return disponibilidad;
	}
	public void setDisponibilidad(boolean disponibilidad) {
		this.disponibilidad = disponibilidad;
	}
	public Persona getVeterinario() {
		return veterinario;
	}
	public void setVeterinario(Persona veterinario) {
		this.veterinario = veterinario;
	}
	
	
	
}

File: Python/gestorAplicacion/Veterinaria/Turno.py
class Turno:
    
    def __init__(self, horaInicio, fecha, mascota, cliente, disponibilidad, veterinario):
        self.horaInicio = horaInicio
        self.fecha = fecha
        self.mascota = mascota
        self.cliente = cliente
        self.disponibilidad = disponibilidad
        self.veterinario = veterinario
        
    def getHoraInicio(self):
        return self.horaInicio
    
    def setHoraInicio(self, horaInicio):
        self.horaInicio = horaInicio
        
    def getFecha(self):
        return self.fecha
    
    def setFecha(self, fecha):
        self.fecha = fecha
        
    def getMascota(self):
        return self.mascota
    
    def setMascota(self, mascota):
        self.mascota = mascota
        
    def getCliente(self):
        return self.cliente
    
    def setCliente(self, cliente):
        self.cliente = cliente
        
    def isDisponibilidad(self):
        return self.disponibilidad
    
    def setDisponibilidad(self, disponibilidad):
        self.disponibilidad = disponibilidad
        
    def getVeterinario(self):
        return self.veterinario
    
    def setVeterinario(self, veterinario):
        self.veterinario = veterinario
        